cmd/cluster/encryption/encryption.go

```diff
@@ -70,6 +70,50 @@ var listCmk = &cobra.Command{
 	},
 }
 
+var updateCmkState = &cobra.Command{
+	Use:   "update-state",
+	Short: "Update Encryption at Rest (EaR) state for a cluster",
+	Long:  "Update Encryption at Rest (EaR) state for a cluster",
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf("Could not initiate api client: %s", err.Error())
+		}
+		authApi.GetInfo("", "")
+		clusterName, _ := cmd.Flags().GetString("cluster-name")
+		clusterId, err := authApi.GetClusterIdByName(clusterName)
+		if err != nil {
+			logrus.Fatalf("%s", ybmAuthClient.GetApiErrorDetails(err))
+		}
+
+		resp, r, err := authApi.ListClusterCMKs(clusterId).Execute()
+		if err != nil {
+			logrus.Debugf("Full HTTP response: %v", r)
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+
+		if resp.Data == (ybmclient.NullableCMKSpec{}) {
+			logrus.Fatalf("No Encryption at rest configuration found for this cluster")
+		}
+
+		cmkId := resp.Data.Get().GetCmkId()
+		newStatus, _ := cmd.Flags().GetBool("enable")
+
+		updateCMKStateSpec := ybmclient.NewUpdateCMKStateSpec(newStatus)
+		resp, r, err = authApi.UpdateClusterCmkState(clusterId, cmkId).UpdateCMKStateSpec(*updateCMKStateSpec).Execute()
+		if err != nil {
+			logrus.Debugf("Full HTTP response: %v", r)
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+
+		state := "DISABLED"
+		if newStatus {
+			state = "ENABLED"
+		}
+		fmt.Printf("Successfully %s updated encryption spec status for cluster %s\n", state, clusterName)
```

Comment:
enabled/disabled based on state - remove updated

Reply:
@arishta-yb you can as well add color as we do in other command

```suggestion
		fmt.Printf("Successfully %s encryption spec status for cluster %s\n, formatter.Colorize(state, formatter.GREEN_COLOR),formatter.Colorize(clusterName, formatter.GREEN_COLOR))

```